test_files/coerce/coerce.js.transformer

```diff
@@ -0,0 +1,12 @@
+goog.module('test_files.coerce.coerce');var module = module || {id: 'test_files/coerce/coerce.js'};/**
```

Comment:
Now that I see the bare files, I see why the patch files were nice.  I hope we are only temporarily in the state where we have transformer/non-transformer output so either seems acceptable to me.